src/log/access_log.cc

```diff
@@ -503,6 +517,15 @@ fvdbDumpTable(StoreEntry * e, hash_table * hash)
     }
 }
 
+static void
+fvdbDumpTable(StoreEntry * e, const HeaderValueCounts& hash)
+{
+    for (const auto & i : hash)
+        storeAppendPrintf(e, "%9lld " SQUIDSBUFPH "\n",
+                        static_cast<unsigned long long>(i.second),
+                        SQUIDSBUFPRINT(i.first));
+}
```

Comment:
Per @rousskov suggestion, please do use the PackableStream feature to produce reports.

```suggestion
fvdbDumpTable(Packable * e, const HeaderValueCounts& hash)
{
    PackableStream os(e);
    for (const auto & i : hash)
        os << std::setw(9) << i.second << " " << i.first << std::endl;
    os.flush();
}
```

Reply:
ok, thanks